CHANGELOG.md

```diff
@@ -0,0 +1,85 @@
+## <small>0.0.1 (2019-05-30)</small>
```

Comment:
No need to generate changelog for the first release

Reply:
Done.